jans-config-api/server/src/main/java/io/jans/configapi/rest/resource/auth/UserResource.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Janssen Project software is available under the MIT License (2008). See http://opensource.org/licenses/MIT for full text.
+ *
+ * Copyright (c) 2020, Janssen Project
+ */
+
+package io.jans.configapi.rest.resource.auth;
+
+import com.github.fge.jsonpatch.JsonPatchException;
+import static io.jans.as.model.util.Util.escapeLog;
+import io.jans.as.common.model.common.User;
+import io.jans.configapi.core.rest.ProtectedApi;
+import io.jans.configapi.rest.model.SearchRequest;
+import io.jans.configapi.service.auth.UsrService;
+import io.jans.configapi.util.ApiAccessConstants;
+import io.jans.configapi.util.ApiConstants;
+import io.jans.configapi.core.util.Jackson;
+import io.jans.orm.model.PagedResult;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.slf4j.Logger;
+
+@Path(ApiConstants.USER)
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@ApplicationScoped
+public class UserResource extends BaseResource {
+
+    private static final String USER = "user";
+
+    @Inject
+    Logger logger;
+
+    @Inject
+    UsrService userSrv;
+
+    @GET
+    @ProtectedApi(scopes = { ApiAccessConstants.USER_READ_ACCESS })
+    public Response getOpenIdConnectClients(
```

Comment:
It's not openid clients, we are returning users. `searchUsers` or `getUsers`

Reply:
Sorry, rectified the same